Cargo.toml

```diff
@@ -11,9 +11,9 @@ keywords = ["ethereum"]
 categories = ["cryptography::cryptocurrencies"]
 
 [dependencies]
-tree_hash = "0.8.0"
+tree_hash = "0.9"
 ethereum_serde_utils = "0.7.0"
-ethereum_ssz = "0.8.0"
+ethereum_ssz = "0.8"
```

Comment:
FYI there's no difference between `0.8.0` and `0.8` version requirements. They both allow semver compatible updates in the patch version

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio